doc/release-notes/index.rst

```diff
@@ -12,12 +12,13 @@ The current Long Term Support (LTS) release for users wanting stability,
 is the OpenQuake Engine 3.16.
 
 The latest stable release for users needing the latest features,
-is the OpenQuake Engine 3.19.
+is the OpenQuake Engine 3.22.
```

Comment:
3.21, no?

Reply:
@pslh perhaps we could also improve the language here to make the distinction between the two releases clearer? 

Right now, we have "... (LTS) release for users wanting stability" vs "latest stable release ...". If both are stable, it might not be clear when to prefer one over the other.

Reply:
I have just removed the qualification "stable"